ECOSYSTEM.md

```diff
@@ -8,6 +8,7 @@ Plugins and tools maintained by the knex team are listed under [Core](#core) whi
 - [`knex-tablecleaner`](https://github.com/kibertoad/knex-tablecleaner) Library for deleting all rows from a given list of DB tables.
 
 #### [Community](#community)
+- [`knemm`](https://github.com/asteinarson/knemm) A CLI tool to manage SQL schema in delcarative YAML files (`claims` and `states`) allowing for dependencies between different loosely coupled modules. It is also a Node.js library.
```

Comment:
delcarative -> declarative
`,` is missing before "allowing"
` It is also a Node.js library.` is redundant